docs/platforms/dart/features/index.mdx

```diff
@@ -20,3 +20,4 @@ Sentry's Dart SDK enables automatic reporting of errors and exceptions, and iden
   - [File I/O Integration](/platforms/dart/integrations/file/).
 - [Logging Integration](/platforms/dart/integrations/logging).
 - [Dio Integration](/platforms/dart/integrations/dio/).
+- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience.
```

Comment:
I'd not add Metrics now, seeing as it's very likely we'll remove or drastically change the API soon